src/composables/transactionToken.ts

```diff
@@ -99,9 +105,70 @@ export const useTransactionToken = (
     || getDexContracts.value.wae.includes(transaction.value.tx.contractId)
   ));
 
+  const labels = computed(() => { //
```

Comment:
1. I suggest to type the returned value as there are multiple returns and there is no easy way to establish this.

2. Unnecessary comment start at the beginning.
3. Rather subjective, but maybe having longer name would be meaningful for this computed property?